docs/_templates/components_requirements.tmpl

```diff
@@ -1,39 +1,76 @@
-This information is not relevant when installing the orchestrator via RPM as is recommended
-in the :ref:`install-server` section.
 
-.. list-table:: Inmanta core and extensions versions
-   :header-rows: 1
+Module compatibility ranges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - .. centered:: Component
-     - .. centered:: Required version
 
-   {% for key, value in data["component_requirements"] | dictsort %}
-   * - .. centered:: {{ key }}
-     - .. centered:: {{ value }}
-   {% endfor %}
+During module development, when the version of the module integrates tightly with a specific ISO version,
+we need a way to tell the CI which version of the module should be used per ISO line. This can be achieved by either:
 
-.. note::
+- Using a specific index per ISO version
+- Adding a project constraint file with the compatible module ranges for each ISO version.
+
+{% set list_only = ["lsm", "std"] %}
+
+.. only:: iso
+
+
+    The table below shows the set of Inmanta modules that integrate tightly with version |version_major| of the
+    Inmanta Service Orchestrator and the constraints on their version.
+
+    .. list-table:: Modules
+       :header-rows: 1
+
+       * - Module
+         - Version constraint
+
+       {% for key, value in data["module_compatibility_ranges"]  | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+         - .. centered:: {{ value }}
+       {% endif %}
+       {% endfor %}
+
+.. only:: oss
+
+    The table below shows the set of Inmanta modules that integrate tightly with this Inmanta OSS version |release|
+    and the constraints on their version.
 
-    This information is also available as a constraints file in `requirements.components.txt <./requirements.components.txt>`_
-    and in json format in the `compatibility.json <./compatibility.json>`_ file.
+    .. list-table:: Modules
+       :header-rows: 1
 
+       * - .. centered:: Module
+
+       {% for key, value in data["module_compatibility_ranges"] | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+       {% endif %}
+       {% endfor %}
+
+.. note::
+
+    This information is also available in a machine-consumable format in the `compatibility.json <./compatibility.json>`_ file.
 
-{% else %}
-Python components are not pinned for major version |version_major| because it is not released yet.
 {% endif %}
 
+
+.. only:: iso
+
+    {% if data['component_requirements'] %}
+
+    .. note::
+
+        If for some reason you need to install any of the components that make up the inmanta-service-orchestrator
+        package separately e.g. when developing an Inmanta extension, the ranges of the Inmanta Python components this
+        product version is compatible with are available as a constraints file in
+        `requirements.components.txt <./requirements.components.txt>`_ and in json format in the
+        `compatibility.json <./compatibility.json>`_ file.
```

Comment:
* This is one big sentence and I find it hard to read. Let's split this sentence into two or more sentences.
* I would first mention the normal procedure here. Namely installing the `inmanta-service-orchestrator~=X.0` to get all the components for a specific ISO version. And then mention that if you want to rely on a specific component(s) only, you need to take into account the following constraints.